libs/components/code-examples/src/lib/modules/forms/radio/icon/example.component.html

```diff
@@ -1,19 +1,17 @@
-<div class="sky-padding-even-lg">
```

Comment:
Removing these wrappers so that the demos don't have excessive padding on the docs site.